addons/account/tests/test_account_bank_statement.py

```diff
@@ -0,0 +1,78 @@
+from odoo.tests import common
+import time
+from odoo import tools
+from odoo.modules.module import get_resource_path
+
+
+class TestAccountBankStatement(common.TransactionCase):
+
+    def test_account_bank_statement_def(self):
+        """  In order to test Bank Statement feature of account I create a bank statement line
+             and confirm it and check it's move created  """
+
+        tools.convert_file(self.cr, 'account',
+                           get_resource_path('account', 'test', 'account_minimal_test.xml'),
+                           {}, 'init', False, 'test', self.registry._assertion_report)
+
+        # Select the period and journal for the bank statement
+        self.journal = self.env['account.bank.statement'].with_context({'lang': u'en_US', 'tz': False, 'active_model': 'ir.ui.menu', 'journal_type': 'bank', 'journal_id': 10, 'date': time.strftime("%Y/%m/%d")})._default_journal()
+        self.assertTrue(self.journal, 'Journal has not been selected')
+
+        # Create a bank statement with Opening and Closing balance 0.
+        self.account_statement = self.env['account.bank.statement'].create({
```

Comment:
There is no need to set anything on the testcase instance, just use local variables.